From a9f5514dcd1dfb5ee605dfb08017818d4c6cc7ce Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Sun, 3 Feb 2019 18:04:29 +0530
Subject: [PATCH] bluetooth: Add changes for vendor complaint

- Define macro to use correct libbt-vendor
- Remove not used file to store bt paired information
- Add overlay with hfp profile disabled

Tracked-On: None
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 groups/bluetooth/btusb/BoardConfig.mk | 2 +-
 groups/bluetooth/btusb/init.rc        | 5 -----
 groups/bluetooth/btusb/product.mk     | 2 +-
 3 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/groups/bluetooth/btusb/BoardConfig.mk b/groups/bluetooth/btusb/BoardConfig.mk
index be3621f..c4d952c 100644
--- a/groups/bluetooth/btusb/BoardConfig.mk
+++ b/groups/bluetooth/btusb/BoardConfig.mk
@@ -1,4 +1,4 @@
-BOARD_HAVE_BLUETOOTH_LINUX_PRI := true
+BOARD_HAVE_BLUETOOTH_INTEL_ICNV := true
 DEVICE_PACKAGE_OVERLAYS += device/intel/common/bluetooth/overlay-bt-pan
 BOARD_SEPOLICY_DIRS += device/intel/project-celadon/sepolicy/bluetooth/common
 
diff --git a/groups/bluetooth/btusb/init.rc b/groups/bluetooth/btusb/init.rc
index d81aced..1bd3ce8 100644
--- a/groups/bluetooth/btusb/init.rc
+++ b/groups/bluetooth/btusb/init.rc
@@ -1,8 +1,3 @@
-on post-fs-data
-    # To store BT paired info
-    mkdir /data/misc/hcid 0770 bluetooth bluetooth
-    setprop persist.bluetooth.enablenewavrcp false
-
 on boot
     chmod 0644 /sys/kernel/debug/bluetooth/l2cap_le_max_credits
     chmod 0644 /sys/kernel/debug/bluetooth/l2cap_le_default_mps
diff --git a/groups/bluetooth/btusb/product.mk b/groups/bluetooth/btusb/product.mk
index c2d7a75..4f987b2 100644
--- a/groups/bluetooth/btusb/product.mk
+++ b/groups/bluetooth/btusb/product.mk
@@ -11,7 +11,7 @@ PRODUCT_PACKAGES += \
   libbt-vendor
 
 {{#ivi}}
-PRODUCT_PACKAGE_OVERLAYS += $(INTEL_PATH_COMMON)/bluetooth/overlay-car
+PRODUCT_PACKAGE_OVERLAYS += $(INTEL_PATH_COMMON)/bluetooth/overlay-car-disablehfp
 {{/ivi}}
 
 {{^ivi}}
-- 
2.17.1

